plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy

```diff
@@ -253,10 +253,17 @@ class SamplesheetConverter {
             
             // Check and convert to the desired format
             def String format = field['value']['format']
-            if(format && format.contains("path")) {
-                def Path inputFile = Nextflow.file(input) as Path
-                return inputFile
+            if(format) {
+                if(format == "file-path-pattern") {
+                    def ArrayList inputFiles = Nextflow.file(input) as ArrayList
+                    return inputFiles
```

Comment:
These files will be added to the channel in an array if I understood well, we should document this